docs/recipes/ec2-eks-metrics-go-adot-ampamg.md

```diff
@@ -1,12 +1,314 @@
 # Using AWS Distro for OpenTelemetry in EKS to ingest metrics into AMP
 
 In this recipe we show you how to instrument a Go application and
-use AWS Distro for OpenTelemetry (ADOT) to ingest metrics into
-Amazon Managed Service for Prometheus (AMP).
-Then we're using AMG to visualize the metrics.
+use [AWS Distro for OpenTelemetry (ADOT)](https://aws.amazon.com/otel) to ingest metrics into
+[Amazon Managed Service for Prometheus (AMP)](https://aws.amazon.com/prometheus/) .
+Then we're using [Amazon Managed Service for Grafana (AMG)](https://aws.amazon.com/grafana/) to visualize the metrics.
+
+We will be setting up an [Amazon Elastic Kubernetes Service (EKS)](https://aws.amazon.com/eks/) cluster and [Amazon Elastic Container Registry (ECR)](https://aws.amazon.com/ecr/) repository to demonstrate a complete scenario.
 
 ## Infrastructure
 
+### Architecture
+
+The ADOT-AMP pipeline enables us to use the ADOT Collector to scrape a Prometheus-instrumented application, and send the scraped metrics to AMP. 
+
+![Architecture](https://aws-otel.github.io/static/Prometheus_Pipeline-07344e5466b05299cff41d09a603e479.png)
+
+The ADOT-AMP pipeline includes two OpenTelemetry Collector components specific to Prometheus — the Prometheus Receiver and the AWS Prometheus Remote Write Exporter. 
+
+[Getting Started with Prometheus Remote Write Exporter for AMP](https://aws-otel.github.io/docs/getting-started/prometheus-remote-write-exporter)
+
+
+### Prerequisites
+
+* The AWS CLI is [installed](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html) and [configured](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html) in your environment.
+* You need to install the [eksctl](https://docs.aws.amazon.com/eks/latest/userguide/eksctl.html) command in your environment.
+* You need to install [kubectl](https://docs.aws.amazon.com/eks/latest/userguide/install-kubectl.html) in your environment. 
+
+### Setup an EKS cluster
+
+For this recipe we require a EKS cluster to be available.
+You can either use an existing EKS cluster or create one using [cluster_config.yaml](ec2-eks-metrics-go-adot-ampamg/cluster_config.yaml).
```

Comment:
Can we please address this issue, cluster_config.yaml does't resolve and results in 404, relative path reference is incorrect.

Reply:
@sheetaljoshi not sure what you mean. It's part of this PR. In order to test it out you need to check out this PR. For example, using the `gh` CLI the command would be `gh pr checkout 5`.

Reply:
@sheetaljoshi Is this still an oustanding issue? Let me know how I can replicate this.

Reply:
http://127.0.0.1:8000/recipes/ec2-eks-metrics-go-adot-ampamg/ec2-eks-metrics-go-adot-ampamg/cluster_config.yaml results into 404. Can you please check.